app/code/Magento/Catalog/Api/CategoryLinkRepositoryInterface.php

```diff
@@ -36,6 +36,19 @@ public function save(\Magento\Catalog\Api\Data\CategoryProductLinkInterface $pro
      */
     public function delete(\Magento\Catalog\Api\Data\CategoryProductLinkInterface $productLink);
 
+    /**
+     * Remove the product assignment from the category by category id and array of sku
+     *
+     * @param int $categoryId
+     * @param array $sku
+     * @return bool will returned True if products successfully deleted
+     *
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     * @throws \Magento\Framework\Exception\StateException
+     * @throws \Magento\Framework\Exception\InputException
+     */
+    public function deleteByIds($categoryId, $sku);
```

Comment:
We can't change API (interfaces) in patch releases. 
It can break a lot of customers.